app-modules/etno/src/Repositories/ItemRepository.php

```
@@ -260,4 +260,24 @@ protected function buildDateFilter(mixed $from, mixed $to): array
             'lte' => \is_numeric($to) ? "{$to}||/y" : $to,
         ]);
     }
+
+    public function incrementProcessingMediaCount(Item $item): void
+    {
+        Cache::increment($this->getProcessingMediaCacheKey($item));
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Support\Facades\Cache' in method 'incrementProcessingMediaCount'. (undefined)

(StaticAccess)